Source/RapyutaSimulationPlugins/Public/Core/RRGameSingleton.h

```diff
@@ -225,17 +233,22 @@ class RAPYUTASIMULATIONPLUGINS_API URRGameSingleton : public UObject
     /**
      * @brief Get the Dynamic Assets Path List object
      * Return the TArray of #SASSET_OWNING_MODULE_NAMES[i] / #DYNAMIC_CONTENTS_FOLDER_NAME
+     * @note #SASSET_OWNING_MODULE_NAMES must have been initialized earlier & once only at Sim start by [ConfigureSimInPlay()]
      * @param InDataType
      * @return TArray<FString>
      */
-    FORCEINLINE static TArray<FString> GetDynamicAssetsPathList(const ERRResourceDataType InDataType)
+    FORCEINLINE static TArray<FString> GetDynamicAssetsBasePathList(const ERRResourceDataType InDataType)
     {
-        static TArray<FString> runtimeAssetsPathList;
-        for (const auto& moduleName : SASSET_OWNING_MODULE_NAMES[InDataType])
+        static TArray<FString> dynamicAssetsBasePathList = [InDataType]()
```

Comment:
what is the main reason to make this lambda function?

Reply:
That is for you to initialize the value once only as the return of an inline lambda call, due to `SASSET_OWNING_MODULE_NAMES` being defined once only at Sim start